src/helpers/news-feed/types.ts

```diff
@@ -0,0 +1,247 @@
+export type EhrNewsFeedFeed = "Immunizations" | "LabReports" | "Procedures" | "Reports";
+
+export interface EhrNewsFeedPageModel {
+    NextPageID?: string
```

Comment:
So these are coming back from the server...but do we want to do the tolowercase() conversion with these? I'm a bit confused by the casing standards we are trying to follow with some of these model objects.

Reply:
Yeah the server has uppercase names for these.  I think not worth the effort to try to coerce them lower just for the JS/TS side of things.  This is not a formally documented API or SDK method; if we did make it one then I think we would tackle that then (or maybe we convert these visualizations to use FHIR...)